lidar_lite_v3hp/CMakeLists.txt

```diff
@@ -0,0 +1,102 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(lidar_lite_v3hp)
+
+## Compile as C++11, supported in ROS Kinetic and newer
+add_compile_options(-std=c++11)
+
+## Find catkin macros and libraries
+## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
+## is used, also find other catkin packages
+find_package(catkin REQUIRED COMPONENTS
+  roscpp
+  message_filters
+  cav_msgs
+  sensor_msgs
+  std_msgs
+)
+
+###################################
+## catkin specific configuration ##
+###################################
+## The catkin_package macro generates cmake config files for your package
+## Declare things to be passed to dependent projects
+## INCLUDE_DIRS: uncomment this if your package contains header files
+## LIBRARIES: libraries you create in this project that dependent projects also need
+## CATKIN_DEPENDS: catkin_packages dependent projects also need
+## DEPENDS: system dependencies of this project that dependent projects also need
+catkin_package(
+#  INCLUDE_DIRS include
+#  LIBRARIES lidar_lite_v3hp
+  CATKIN_DEPENDS roscpp message_filters cav_msgs sensor_msgs std_msgs
+#  DEPENDS system_lib
+)
+
+###########
+## Build ##
+###########
+
+## Specify additional locations of header files
+## Your package locations should be listed before other locations
+include_directories(
+  include
+  ${catkin_INCLUDE_DIRS}
+)
+
+add_executable(lidar_litev3hp_node src/main.cpp src/lidar_litev3hp_wrapper.cpp src/lidar_litev3hp_worker.cpp)
+
+add_library(lidar_litev3hp_worker_lib src/lidar_litev3hp_worker.cpp)
+add_dependencies(lidar_litev3hp_worker_lib ${catkin_EXPORTED_TARGETS})
+
+## Specify libraries to link a library or executable target against
+target_link_libraries(lidar_litev3hp_node ${catkin_LIBRARIES})
+
+
+#############
+## Install ##
+#############
+
```

Comment:
Every package you make must have install tags. CARMA runs by sourcing install/setup.bash not devel.